isthmus/src/test/java/io/substrait/isthmus/ArithmeticFunctionTest.java

```diff
@@ -16,62 +16,97 @@ void arithmetic(String c) throws Exception {
     String query =
         String.format(
             "SELECT %s + %s, %s - %s, %s * %s, %s / %s FROM numbers", c, c, c, c, c, c, c, c);
-    assertSqlSubstraitRelRoundTrip(query, CREATES);
+    assertFullRoundTrip(query, CREATES);
```

Comment:
`assertFullRoundTrip` is a strong assert. Will eventually remove all instances of `assertSqlSubstraitRelRoundTrip`